pop-up-details-desktop.js

```diff
@@ -0,0 +1,409 @@
+const bigContainer = document.createElement('div');
+bigContainer.setAttribute('class', 'projectPopUp');
+bigContainer.style.width = '100%';
+bigContainer.style.height = '1851px';
+
+const insideProject = document.createElement('div');
+insideProject.setAttribute('class', 'inside');
+insideProject.style.width = '100%';
+insideProject.style.height = '1711px';
+
+document.body.appendChild(bigContainer);
+bigContainer.append(insideProject);
+
+const myFirstProjectBtn = document.querySelector('.my-first-project-button');
+const mySecondProjectBtn = document.querySelector('.my-second-project-button');
+
+const rectBar = document.createElement('span');
+const rectBar2 = document.createElement('span');
+
+const projectName = document.createElement('h2');
+projectName.setAttribute('class', 'projectTitle');
+projectName.innerHTML = 'Project name goes here';
+
+const progLang = document.createElement('ul');
+progLang.setAttribute('class', 'stackList');
+
+const progLangList1 = document.createElement('li');
+progLangList1.setAttribute('class', 'list1');
+progLangList1.innerHTML = 'HTML/CSS';
+
+const progLangList2 = document.createElement('li');
+progLangList2.setAttribute('class', 'list2');
+progLangList2.innerHTML = 'Ruby on Rails';
+
```

Comment:
According to the project requirements, you are expected to create an array of objects for each project details and instead of manually creating the popups, you could loop through that array to create the modal dynamically in order to satisfy the project requirements.

Your array should look similar to this:
```
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

Reply:
- [ ] As indicated by previous reviewer, I cannot find where you have created the array of objects. It seems you have not implemented that change at all. Kindly do so.

According to the project requirements, you are expected to create an array of objects for each project details and instead of manually creating the popups, you could loop through that array to create the modal dynamically in order to satisfy the project requirements.

Your array should look similar to this:
```
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```